web/controllers/certificateController/index.ts

```diff
@@ -87,6 +89,8 @@ export async function createCertificateEndpoint(req: Request, res: Response) {
         if (requestor.wix_id != req.params.student)
             return res.status(403).send("Students may only retrieve certificates for themselves");
 
+        let state = req.params.automatic ? State.awaitingApproval : State.manual;
```

Comment:
So having this information in the URL parameters, instead of including it in the query parameters, means that we also need to change the definition of the endpoint. Or should that be included later and disabled for now?

Reply:
Ah, right, not sure why I chose `req.params` here ... `req.query` makes much more sense